src/strava_offline/cli.py

```diff
@@ -65,6 +70,27 @@ def cli_report_yearly_bikes(config: config.DatabaseConfig, output: TextIO, year:
     with sync.database(config) as db:
         print(reports.yearly_bikes(db, year), file=output)
 
+@cli.command(name='report-bikes-cummulative-distance')
+@config.DatabaseConfig.options()
+@option_output
+@option_time_resolution
+@option_output_format
+@option_bike
+@option_year_start
+@option_year_end
+def cli_report_bikes_cummulative_distance(
+        config: config.DatabaseConfig,
+        output: TextIO,
+        resolution: str,
+        format: str,
+        bike: Optional[str],
+        start_year: int, end_year: int,
+) -> None:
+    "Show monthly report by bike"
```

Comment:
It's actually daily, isn't it? Also probably wanna mention "cummulative" here as well :-)

Reply:
Yes! I probably changed the default and completely forgot about the docstr